<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>颜色转移</string>
  </property>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>579</y>
     <width>960</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnLoadImg">
      <property name="text">
       <string>选取待上色图像</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadRef">
      <property name="text">
       <string>选取参考图像</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadFile">
      <property name="text">
       <string>读入选区文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnColorize">
      <property name="text">
       <string>上色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveImg">
      <property name="text">
       <string>保存图像</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>10</y>
     <width>16</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>460</width>
     <height>560</height>
    </rect>
   </property>
   <property name="title">
    <string>参考图像</string>
   </property>
   <widget class="QLabel" name="refLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>460</width>
     <height>560</height>
    </rect>
   </property>
   <property name="title">
    <string>待上色图像</string>
   </property>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
